Week-2/Assignment_2.2/assets/CSS/style.css

```diff
@@ -0,0 +1,56 @@
+/* Set up 4 boxes with different colour and parent container */
```

Comment:
is and where css functions can be used for same css for multiple classes https://developer.mozilla.org/en-US/docs/Web/CSS/:where